2-With-broker/UserDetailsClient/UserDetailsClient/App.cs

```diff
@@ -13,6 +13,7 @@ public class App : Application
         /// </summary>
         public static string ClientID = "4a1aa1d5-c567-49d0-ad0b-cd957a47f842";
         public const string BrokerRedirectUriOnIos = "msauth.com.yourcompany.UserDetailsClient://auth";
+        public const string BrokerRedirectUriOnAndroid = "msauth://UserDetailsClient.Droid/{Your package signature}";
```

Comment:
how does the package signature get replaced or is that a manual step?

Reply:
This will need to be a manual step unfortunately

Reply:
adding additional steps